### PR TITLE
Create display mode change and fix minor issues

### PR DESCRIPTION
**Pull request by midasdev711**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,18 +8,23 @@
...
     "chalk": "^4.1.1",
+    "classnames": "^2.3.1",
     "console-log-level": "^1.4.1",
     "cors": "^2.8.5",
     "cross-fetch": "^3.1.4",
     "ethers": "^5.4.2",
     "express": "^4.17.1",
+    "forever": "^4.0.1",
+    "framer-motion": "^4.1",
+    "got": "^11.8.2",
```
Comment (gdev8317) [inline package.json:27]: i guess `got` and `forever` are not used anymore (at least i removed them in `master` branch). if you don't need the let's remove these deps

File: src/Arbitrum.js [hunk 2]
```diff
@@ -166,9 +173,9 @@ function Arbitrum() {
 
   return (
     <div className="Home">
-      <h1>GMX Analytics / Arbitrum</h1>
+      <h1>Analytics / Arbitrum</h1>
       {lastSubgraphBlock && lastBlock &&
-        <p className={isObsolete ? 'warning' : ''} style={{ marginTop: '-1rem' }}>
+        <p className={cx('page-description', (isObsolete ? 'warning' : ''))} style={{ marginTop: '-1rem' }}>
```
Comment (gdev8317) [inline src/Arbitrum.js:178]: with `classnames` it's possible to write `cx('page-description', {warning: isObsolete})`, maybe a bit more convenient
fyi

File: src/helpers.js [hunk 3]
```diff
@@ -11,8 +11,24 @@ const { BigNumber } = ethers
...
+//   '#00BFEA',
+//   '#949FE1',
+//   '#CC61B0',
+//   '#c90000',
+//   '#7b7b7b',
+//   '#6464ff',
+//   'purple',
+//   'darkgreen',
+//   RED
+// ]
```
Comment (gdev8317) [inline src/helpers.js:31]: do we need this commented code? if no let's remove it

File: src/Bsc.js [hunk 4]
```diff
@@ -332,12 +332,12 @@ function Bsc() {
...
-              <Bar type="monotone" unit={dynamicUnit} dataKey="BUSD" stackId="a" name="BUSD" fill="#c90000" />
-              <Bar type="monotone" unit={dynamicUnit} dataKey="BTC" stackId="a" name="BTC" fill="#3483eb" />
-              <Bar type="monotone" unit={dynamicUnit} dataKey="ETH" stackId="a" name="ETH" fill="#eb8334" />
-              <Bar type="monotone" unit={dynamicUnit} dataKey="BNB" stackId="a" name="BNB" fill="#ee64b8" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="USDC" stackId="a" name="USDC" fill="#2775CA" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="USDT" stackId="a" name="USDT" fill="#26A17B" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="BUSD" stackId="a" name="BUSD" fill="#F0B90B" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="BTC" stackId="a" name="BTC" fill="#FF9800" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="ETH" stackId="a" name="ETH" fill="#627EEA" />
+              <Bar type="monotone" unit={dynamicUnit} dataKey="BNB" stackId="a" name="BNB" fill="#FE88B1" />
```
Comment (gdev8317) [inline src/Bsc.js:340]: i know i've hardcoded all those colors initially
in future if you'll work with something related again would be good to unhardcode them and use variables/color config as in other places

thanks :)

File: src/Arbitrum.js (hunk 2 above)
Comment (midasdev711) [reply src/Arbitrum.js]: Fixed.

File: package.json (hunk 1 above)
Comment (midasdev711) [reply package.json]: Removed got and forever.

File: src/helpers.js (hunk 3 above)
Comment (midasdev711) [reply src/helpers.js]: Removed

File: src/Bsc.js (hunk 4 above)
Comment (midasdev711) [reply src/Bsc.js]: Done

**Pull request merged**